charts/cluster/values.schema.json

```diff
@@ -213,6 +213,9 @@
                             "properties": {
                                 "enabled": {
                                     "type": "boolean"
+                                },
+                                "excludeRules": {
+                                    "type": "array"
```

Comment:
We could define the array to be of strings and define all the possible values, but maybe it's a bit overkill

Reply:
The moment we start doing that we have to throw away the schema generator, because it will overwrite the customisations.

Reply:
Not that I love it, but it does offer some convenience.

Reply:
Yeah, I know, it would be nice to have annotations similar to kubebuilder, but I'm not sure I'll have any time to work on it in the foreseeable future 😅